_rest_routes.inc.php

```diff
@@ -345,6 +346,14 @@
         RestConfig::authorization_check("patients", "notes");
         return (new MessageRestController())->delete($pid, $mid);
     },
+    "GET /api/immunization" => function () {
+        RestConfig::authorization_check("admin", "users");
```

Comment:
```
RestConfig::authorization_check("patients", "med");
```
reference:
https://www.open-emr.org/wiki/index.php/Access_Controls_Listing#Medical_Records_and_History_.28write.2Caddonly_optional.29_.28med.29

Reply:
Thanks for sharing the reference :)